specs/dwritecore/DWriteCore.md

```diff
@@ -78,33 +78,272 @@ with interoperability issues.
 New APIs introduced by DWriteCore include:
 
  - The exported function is renamed `DWriteCoreCreateFactory` to avoid a naming collision with the
-   `DWriteCreateFactory` function exported by the Windows DLL (DWrite.dll).
- - A new DWRITE_FACTORY_TYPE enumeration for creating an isolated factory object
+   `DWriteCreateFactory` function exported by the system version of DirectWrite (DWrite.dll)
+ - A new `DWRITE_FACTORY_TYPE` enumeration value for creating an isolated factory object
```

Comment:
Nit: DWRITE_FACTORY_TYPE here is an _enumeration_. It contains a number of _enumerators_.

```suggestion
 - A new `DWRITE_FACTORY_TYPE` enumeration for creating an isolated factory object
```